test/reuse/helper/errorHandler/logException.spec.js

```diff
@@ -0,0 +1,85 @@
+"use strict";
+
+const ErrorHandler = require("../../../../lib/reuse/helper/errorHandler").default;
+
+describe("errorHandler - logException - displaying customized message without stacktrace", async function () {
+  let errorMessage;
+  it("Execution", async function () {
+    const errorHandler = new ErrorHandler(false);
+    try {
+      throw new Error("Divide by zero error.");
+    } catch (error) {
+      try {
```

Comment:
Instead of catching the thrown error you can also use something like:

```js
await expect(errorHandler.logException(error))
      .rejects.toThrow("Error: Function '' failed with : Divide by zero error."");
```

Reply:
done required changes